sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +94,99 @@ TypeScript users need to have Node type definitions installed:
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyVaultBackupClient } from "@azure/keyvault-admin";
+
+const credentials = new DefaultAzureCredential();
+
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const client = new KeyVaultBackupClient(vaultUrl, credentials);
+```
 
 ## Key concepts
 
-## Authenticating with Azure Active Directory
+### KeyVaultRoleDefinition
+
+A Role Definition is a collection of permissions. A role definition defines the operations that can be performed, such as read, write, 
+and delete. It can also define the operations that are excluded from allowed operations.
+
+Role definitions can be listed and specified as part of a Role Assignment.
+
+### KeyVaultRoleAssignment. 
+
+A Role Assignment is the association of a Role Definition to a service principal. They can be created, listed, fetched individually, and deleted.
+
+### KeyVaultAccessControlClient
+
+A [KeyVaultAccessControlClient][src-rbac-client] provides both synchronous and asynchronous operations allowing for management of Role Definitions (instances of `KeyVaultRoleDefinition`) and Role Assignments (instances of `KeyVaultRoleAssignment`).
+
+### KeyVaultBackupClient
+
+A [KeyVaultBackupClient][src-backup-client] provides both synchronous and asynchronous operations for performing full key backups, full key restores, and selective key restores.
+
+### Long running operations
+
+The Azure SDK for TypeScript and JavaScript offers a common abstraction for long running operations through the package [core-lro][core-lro]. In particular, the operations done by the [KeyVaultBackupClient][src-backup-client] may take as much time as needed by the Azure resources, requiring a client layer to keep track, serialize and resume the operations through the lifecycle of the programs that wait for them to finish.
+
+The [KeyVaultBackupClient][src-backup-client] offers three methods that execute long running operations:
+
+- `beginBackup`, starts generating a backup of an Azure Key Vault on the specified Storage Blob account.
+- `beginRestore`, starts restoring all key materials using the SAS token pointing to a previously stored Azure Blob storage backup folder.
+- `beginSelectiveRestore`, starts restoring all key versions of a given key using user supplied SAS token pointing to a previously stored Azure Blob storage backup folder.
+
+The methods that begin long running operations return a poller that allows you to wait indefinitely until the operation is complete. More information is available on the examples below.
 
 ## Examples
 
+We have provided readmes with the detailed steps to run our samples, both for [JavaScript](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/javascript/README.md) and [TypeScript](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/typescript/README.md). Direct links to the specific samples follow:
```

Comment:
The link texts `JavaScript` and `TypeScript` do not have enough context when read through screenreaders

```suggestion
We have samples both in JavaScript and TypeScript that show the access control and backup/restore features in this package. Please follow the corresponding readmes for detailed steps to run the samples.
- [Readme for JavaScript samples](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/javascript/README.md)
- [Readme for TypeScript samples](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/typescript/README.md)

Direct links to the specific samples follow:
```

Reply:
Awesome feedback! thank you!